### PR TITLE
fix(snyk): fix dependencies version from snyk

### PR DESCRIPTION
**Pull request by Sanjay-Veernala**
## Description of the change
- added all version upgrades done by snyk in this PR
> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by koladilip — DISMISSED** [review]

**Review by shrouti1507 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -51,10 +51,11 @@
   },
   "dependencies": {
     "@amplitude/ua-parser-js": "^0.7.24",
-    "@aws-sdk/client-personalize": "^3.363.0",
+    "@aws-sdk/client-personalize": "^3.391.0",
+    "@aws-sdk/client-lambda": "^3.276.0",
```
Comment (krishna2020) [inline package.json:55]: what for is this new dependency `client-lambda` ?

Comment (Sanjay-Veernala) [reply package.json:55]: This dependency was in the snyk fix open PR.
So, I have added . if not needed can remove this .

File: package.json [hunk 2]
```diff
@@ -64,6 +65,7 @@
     "ajv": "^8.12.0",
     "ajv-draft-04": "^1.0.0",
     "ajv-formats": "^2.1.1",
```
Comment (krishna2020) [inline package.json:67]: check on `aws-sdk` & `pprof` new dependencies too

Comment (krishna2020) [reply package.json:67]: pprof dependency has been long ago removed & instead we are using "@datadog/pprof": "^3.1.0" now - share the PR that suggested this change

File: package.json (hunk 1 above)
Comment (krishna2020) [reply package.json:55]: can you share the PR to know whether to keep or remove it

File: package.json (hunk 2 above)
Comment (Sanjay-Veernala) [reply package.json:67]: https://github.com/rudderlabs/rudder-transformer/pull/2498/files

Comment (krishna2020) [reply package.json:67]: These are old PR's after which few dependencies got cleaned up - https://github.com/rudderlabs/rudder-transformer/commit/397277b4328f94ed6b732a2f61677eaba5e7bda0
Remove the new dependencies added in this PR

File: package.json [hunk 3]
```diff
@@ -64,6 +64,7 @@
...
+    "aws-sdk": "^2.1354.0",
```
[the remark this answers is not in the document]
Comment (Sanjay-Veernala) [reply package.json]: removed

**Review by krishna2020 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**